AztecTests/HTML/ElementNodeTests.swift

```diff
@@ -691,7 +692,7 @@ class ElementNodeTests: XCTestCase {
         
         let wrapRange = NSRange(location: 0, length: fullText.characters.count)
         
-        paragraph.forceWrap(range: wrapRange, inNodeNamed: "b", withAttributes: [], equivalentElementNames: [])
+        paragraph.forceWrap(range: wrapRange, inElement: ElementNodeDescriptor(name: "b"))
```

Comment:
We could use this PR to refactor the use of "b" for StandardElementType.b.rawValue. Just to get rid of this loose strings :)